providers/ns1/ns1Provider.go

```diff
@@ -192,16 +197,22 @@ func convert(zr *dns.ZoneRecord, domain string) ([]*models.RecordConfig, error)
 		case "ALIAS":
 			rec.Type = rtype
 			if err := rec.SetTarget(ans); err != nil {
-				panic(fmt.Errorf("unparsable %s record received from ns1: %w", rtype, err))
+				return found, fmt.Errorf("unparsable %s record received from ns1: %w", rtype, err)
```

Comment:
When returning an error, the other parameters should be nil or zero.

```suggestion
				return nil, fmt.Errorf("unparsable %s record received from ns1: %w", rtype, err)
```

(Same for the other cases)